opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -644,8 +646,11 @@ message DoubleExemplar {
   // 1970.
   fixed64 time_unix_nano = 2;
 
-  // Numerical double value of the measurement that was recorded.
-  double value = 3;
+  // Numerical value of the measurement that was recorded.
+  oneof value {
```

Comment:
nit: Similar to #278, let's say the point is invalid when no value is present.

Reply:
Done.